src/components/BoxForm/BoxForm.jsx

```diff
@@ -0,0 +1,232 @@
+import React, { useState, useEffect } from 'react';
+import {
+  FormErrorMessage,
+  Textarea,
+  FormControl,
+  FormLabel,
+  Input,
+  Checkbox,
+  Button,
+} from '@chakra-ui/react';
+import { InfoIcon } from '@chakra-ui/icons';
+import axios from 'axios';
+import DatePicker from 'react-datepicker';
+import { FYABackend, isValidZip } from '../../common/utils';
+import DropZone from './DropZone/DropZone';
+import 'react-datepicker/dist/react-datepicker.css';
+import './BoxForm.css';
+import './DatePicker.css';
+
+function Box() {
```

Comment:
Change component to arrow function to follow ES6 conventions and so that we can keep things consistent throughout